### ISSUE TITLE
Support OCaml 4.08

### ISSUE THREAD

**Issue by Lupus** [jaredly/reason-language-server#317]
Looks like 4.08.0 is not supported on 1.7.1. Here's what I get on freshly created opam switch with 4.08.0 compiler:

```
[Error - 1:30:27 PM] Request textDocument/hover failed.
  Message: Unsupported OCaml version: 4.08.0
  Code: -32603 
```

**Comment by Lupus** [comment]
Oh, looks like it's nearly solved in PR #305 ! Looking forward to it :)

**Comment by Lupus** [comment]
It's nearly 3 months since last release. We have to distribute locally built extension from master to have 4.08 support, which is far from convenient... @jaredly are there any plans to cut a new release with recent improvements?

**Comment by swuecho** [comment]
any update? Thanks.

**Comment by jaredly** [comment]
I just had my first baby, sorry for the delay in open source things ;) I'll try to get a release out in the next couple of days

**Comment by jaredly** [comment]
Ok, published! 1.7.2
